tools/build-manylinux.sh

```diff
@@ -33,7 +33,7 @@ if [[ ! -f /.dockerenv ]]; then
         exit 1
     fi
 
-    docker run -t -v $(pwd):/io quay.io/pypa/manylinux2010_x86_64:latest  /io/tools/build-manylinux.sh "$LIBRDKAFKA_VERSION"
+    docker run -t -v $(pwd):/io quay.io/pypa/manylinux_2_28_aarch64:latest  /io/tools/build-manylinux.sh "$LIBRDKAFKA_VERSION"
```

Comment:
Since this script is specific to arm64 build, I think this change should be done in arm64 specific script. Keep this file as it is.

Is this file getting used anywhere else as well?

Reply:
Not getting used elsewhere. I've updated to include an if else check based on the ARCH variable set to pick up the correct image.